adage/analyze/management/commands/import_gene_sample_expr.py

```diff
@@ -109,17 +136,21 @@ def import_data_line(line_num, tokens, samples):
 
     gene_name = tokens[0]
     if not gene_name or gene_name.isspace():
-        raise Exception("Input file line #%d: gene systematic_name (column #1)"
+        raise Exception("Input file line #%d: gene name (column #1)"
                         " is blank" % line_num)
 
     try:
-        gene = Gene.objects.get(systematic_name=gene_name)
+        gene = Gene.objects.get(systematic_name=gene_name, organism=organism)
+    except Gene.MultipleObjectsReturned:
+        raise Exception("Input file line #%d: gene name %s (column #1) matches"
+                        " multiple genes in the database" %
+                        (line_num, gene_name))
     except Gene.DoesNotExist:
         # If a gene is not found in database, generate a warning message
         # and skip this line.
         logger.warn(
-            "Input file line #%d: gene systematic_name (column #1) is not "
-            "found in the database: %s", line_num, gene_name)
+            "Input file line #%d: gene name %s (column #1) not found in "
+            "database", line_num, gene_name)
```

Comment:
Maybe I'm missing something, but don't we need this to say:

```python
...database" % (line_num, gene_name))
```

(as in lines 145-147 above) and I think line 151 also needs to be `logger.warning(`, right?